docs/social-messaging/index.md

```diff
@@ -223,11 +223,11 @@ The Social Messaging event notification supports the following event types:
 
 ### Identities API
 
-The Identity APIs facilitate the reading of customer and agent identities from a RingCX Digital account.
+The Identity APIs facilitate the reading of customer and agent identities from a RingEX Digital account.
```

Comment:
Same here... Should we call it as just "RingEX account" instead of "RingEX Digital account"?